proto/cusf/mainchain/v1/wallet.proto

```diff
@@ -100,3 +107,47 @@ message GenerateBlocksRequest {
 
 message GenerateBlocksResponse {
 }
+
+message GetBalanceRequest {
+}
+message GetBalanceResponse {
+  uint64 confirmed_satoshi = 1;
+  uint64 pending_satoshi = 2;
+}
+
+message TransactionConfirmation {
```

Comment:
Top-level message types should be defined before services. Please move this to before services, or move it within request/response messages.